templates/packages.dhall

```diff
@@ -118,4 +118,4 @@ let overrides = {=}
 
 let additions = {=}
 
-in  upstream ⫽ overrides ⫽ additions
+in  upstream // overrides // additions
```

Comment:
This is the simplest workaround for the unicode problem, using ASCII in the template here. Users can still use unicode in the file once it's been created, this workaround keeps the UX nice (until a better solution is found).

Reply:
That works perfectly :)

Reply:
Awesome! I'll merge that in then, so we can just rebase this one :)